sdk/flow/confirm.go

```diff
@@ -51,7 +52,8 @@ func (item *ConfirmerItem) Confirm(ctx *cli.Context) (bool, error) {
 	fmt.Scanln(&option)
 	action, err := item.action(option)
 	if err != nil {
-		return false, errors.Trace(err)
+		fmt.Fprintf(os.Stderr, "no action found for option %q", option)
```

Comment:
We should check the error string to make sure it's due to a typo.

Reply:
I'm pretty sure this is the only possible error returned from that func.